bob/src/cluster/operations.rs

```diff
@@ -61,10 +84,11 @@ async fn finish_all_handles<TErr: Debug>(
 ) -> Vec<NodeOutput<TErr>> {
     let mut ok_count = 0;
     let mut total_count = 0;
+    let mut oks = Vec::new();
```

Comment:
Most of the time all `handles` will complete with success, so I suggest to create this vector with initial capacity = `handles.len()`